index.html

```diff
@@ -4049,6 +4081,88 @@ <h2>
+          the user is interacting with.
+          </li>
+          <li>If <var>request</var>.<a>[[\response]]</a> is null, terminate
+          this algorithm.
+          </li>
+          <li>Let <var>response</var> be
+          <var>request</var>.<a>[[\response]]</a>.
+          </li>
+          <li>
+            <a>Queue a task</a> on the <a>user interaction task source</a> to
+            run the following steps:
+            <ol data-link-for="PaymentResponse">
+              <li>Assert: <var>request</var>.<a>[[\updating]]</a> is false.
+              </li>
+              <li>Assert: <var>request</var>.<a>[[\state]]</a> is
+              "<a>interactive</a>".
+              </li>
+              <li>If <var>payer name</var> changed:
+                <ol>
+                  <li data-link-for="PaymentOptions">Assert:
+                  <var>request</var>.<a>[[\options]]</a>.<a>requestPayerName</a>
+                  is true.
+                  </li>
+                  <li>Set <var>response</var>'s <a>payerName</a> attribute to
+                  <var>payer name</var>.
+                  </li>
+                </ol>
+              </li>
+              <li>If <var>payer email</var> changed:
+                <ol>
+                  <li data-link-for="PaymentOptions">Assert:
+                  <var>request</var>.<a>[[\options]]</a>.<a>requestPayerEmail</a>
+                  is true.
+                  </li>
+                  <li>Set <var>response</var>'s <a>payerEmail</a> attribute to
+                  <var>payer email</var>.
+                  </li>
+                </ol>
+              </li>
+              <li>If <var>payer phone</var> changed:
+                <ol>
+                  <li data-link-for="PaymentOptions">Assert:
+                  <var>request</var>.<a>[[\options]]</a>.<a>requestPayerPhone</a>
+                  is true.
+                  </li>
+                  <li>Set <var>response</var>'s <a>payerPhone</a> attribute to
+                  <var>payer phone</var>.
+                  </li>
+                </ol>
+              </li>
+              <li>Let <var>event</var> be the result of <a data-cite=
+              "!DOM#concept-event-create">creating an event</a> using
+              <a>PaymentRequestUpdateEvent</a>.
+              </li>
+              <li>Initialize <var>event</var>'s <code><a data-cite=
+              "!DOM#dom-event-type">type</a></code> attribute to
+              "<a>payerdetailchange</a>".
+              </li>
+              <li>
+                <a data-cite="!DOM#concept-event-dispatch">Dispatch</a>
+                <var>event</var> at <var>response</var>.
+              </li>
+              <li data-link-for="PaymentRequestUpdateEvent">If
+              <var>event</var>.<a>[[\waitForUpdate]]</a> is true, disable any
+              part of the user interface that could cause another update event
+              to be fired.
```

Comment:
Is "another update event" appropriate here? Maybe "another change to the payer details"?